chapters/ch07.asciidoc

```diff
@@ -1419,9 +1419,9 @@ const things = {
   length: 2
 }
 Array.from(things)
-// <- [<div>, <div>, <div>, ...]
+// <- [Object, Object]
```

Comment:
It does, but we are showing that the resulting array contains the same object reference as the `groceries` object does. Using destructuring here wouldn't show that